src/server/session.ts

```diff
@@ -550,11 +550,11 @@ module ts.server {
             }
 
             return completions.entries.reduce((result: protocol.CompletionEntry[], entry: ts.CompletionEntry) => {
-                if (completions.isMemberCompletion || entry.name.indexOf(prefix) == 0) {
+                if (completions.isMemberCompletion || (entry.name.toLowerCase().indexOf(prefix.toLowerCase()) == 0)) {
                     result.push(entry);
                 }
                 return result;
-            }, []);
+            }, []).sort((a, b) => a.name.localeCompare(b.name));

```

Comment:
sort should be alphabetical but not case sensitive, `a.name.toLowerCase().localeCompare(b.name.toLowerCase())`